.github/workflows/release-body-driver.yml

```diff
@@ -95,8 +95,6 @@ jobs:
       # Finally, merge them together
       - name: Append matrixes to create release body
         run: |
-          mv matrix_${{ github.event.release.tag_name }}_X64.md matrix_X64.md
-          mv matrix_${{ github.event.release.tag_name }}_ARM64.md matrix_ARM64.md
```

Comment:
don't we need a `mv` here?

Reply:
This was just a rename, but matrixes inside downmload artifacts are already named like that:
> path: ./matrix_gen/matrix_${{matrix.architecture}}.md

Reply:
ok that's true thanks!